### ISSUE TITLE
Is there any plans on GlowFilter and DropShadowFilter?

### ISSUE THREAD

**Issue by sky0014** [CreateJS/EaselJS#943]
Just like in Flash:

https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/filters/GlowFilter.html
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/filters/DropShadowFilter.html

It's very useful, hope involved, thank you.

**Comment by DavidHGillen** [comment]
Yes, They're on the list of TODO's on my desk right now. They're derivatives of a blur filter so you can implement them yourself in the mean time (if you're not sure how, feel free to ask). We've been a little heads down on some major internal pipeline updates for EaselJS but as those things are starting to wrap up some more filters are definitely planned. You can have a peek at a couple of the new commers here:
https://github.com/DavidHGillen/EaselJS/blob/master/src/easeljs/filters/AberrationFilter.js
https://github.com/DavidHGillen/EaselJS/blob/master/src/easeljs/filters/DisplacementFilter.js

**Comment by gskinner** [comment]
Dropshadow and glow are already supported in `Stage` (@DavidHGillen is talking about `StageGL`) via `myDisplayObject.shadow`. Note that a glow is just a shadow with no offset.

**Comment by sebastianderossi** [comment]
@sky0014 This demo should help :) https://codepen.io/createjs/full/c11bb6183ed1248440aeb5aa68a89e7d/

**Comment by sky0014** [comment]
@DavidHGillen thank you for your reply. Actually I modified a third-party library to use for now( https://github.com/u-kudox/Filters_for_EaselJS ), But I think it's better for you guys to implement it. 😄

**Comment by sky0014** [comment]
@gskinner @sebastianderossi Thank you for the demo : )
I tried shadow and stroke to make some effect like GlowFilter does in Flash. But I found that there's some tiny difference. I used with Text, my parameters is `GlowFilter(0,1,2,2,3)`. I found it's hard to expand four directions. If I draw with lager font-size to simulate shadow, position may be dislocation.

**Comment by normmcgarry** [comment]
@sky0014 Any chance you could share your modifications to that library?
